backend/src/v5/processors/teamspaces/projects/models/drawings.js

```diff
@@ -35,7 +40,48 @@ Drawings.addDrawing = (teamspace, project, data) => addModel(teamspace, project,
 
 Drawings.updateSettings = updateModelSettings;
 
-Drawings.deleteDrawing = deleteModel;
+Drawings.deleteDrawing = async (teamspace, project, drawing) => {
+	await removeFilesWithMeta(teamspace, DRAWINGS_HISTORY_REF_COL, { model: drawing });
+
+	await Promise.all([
+		deleteModelRevisions(teamspace, project, drawing, modelTypes.DRAWING),
+		deleteModel(teamspace, project, drawing),
+		removeModelFromProject(teamspace, project, drawing),
+	]);
+};
+
+Drawings.getRevisions = async (teamspace, drawing, showVoid) => {
+	const revisions = await getRevisions(teamspace, drawing, modelTypes.DRAWING, showVoid,
```

Comment:
Can just remove async from function declaration and do this
```suggestion
	return getRevisions(teamspace, drawing, modelTypes.DRAWING, showVoid,
```